numpyro/hmc_util.py

```diff
@@ -567,6 +567,24 @@ def transform_fn(transforms, params, invert=False):
 
 
 def initialize_model(rng, model, *model_args, **model_kwargs):
+    """
+    Given a model with Pyro primitives, returns a function which, given
+    unconstrained parameters, evaluates the potential energy (negative
+    joint density). In addition, this also returns initial parameters
+    sampled from the prior to initiate MCMC sampling and functions to
+    transform unconstrained values at sample sites to constrained values
+    within their respective support.
+
+    :param jax.random.PRNGKey rng: random number generator seed to
+        sample from the prior.
+    :param model: Python callable containing Pyro primitives.
+    :param tuple model_args: args provided to the model.
+    :param dict model_kwargs: kwargs provided to the model.
+    :return: tuple of (`init_params`, `potential_fn`, `transforms`)
+        `init_params` are values from the prior used to initiate MCMC.
+        `transforms` are useful to convert unconstrained HMC samples
```

Comment:
typo: `transform_fn` instead of `transforms`. How about we rename it to `inv_transform_fn` to be aligned with Pyro NUTS's `transforms` and Stan reference manual?

Reply:
Sounds good to me.